onadata/apps/logger/migrations/0030_backfill_lost_monthly_counters.py

```diff
@@ -0,0 +1,74 @@
+from django.db import migrations
+from django.db.migrations.recorder import MigrationRecorder
+from django.db.models import Sum
+from django.db.models import Value, F, DateField
+from django.db.models.functions import Cast, Concat
+from django.db.models.functions import ExtractYear, ExtractMonth
+from django.utils.timezone import now
```

Comment:
Nitpick: I prefer importing `timezone` and calling `timezone.now()`, which is unambiguous